justfile

```diff
@@ -32,9 +32,7 @@ _rustflags := env_var_or_default("RUSTFLAGS", "")
 
 # If we're running in Github Actions and cargo-action-fmt is installed, then add
 # a command suffix that formats errors.
-#
-# Clippy version also gets -Dwarnings.
-_fmt_clippy := if env_var_or_default("GITHUB_ACTIONS", "") != "true" { "-- -Dwarnings" } else {
+_fmt_clippy := if env_var_or_default("GITHUB_ACTIONS", "") != "true" { "" } else {
```

Comment:
Do we have some way of handling this to actually deny clippy warnings in CI?

Reply:
if `GITHUB_ACTIONS=true` it goes into the `else`, which is what sets `-Dwarnings`

Reply:
it's the case where `GITHUB_ACTIONS != "true"` where we no longer deny warnings

Reply:
ah, thank you, lacking reading comprehension.